pkg/api/deployment.go

```diff
@@ -34,21 +35,51 @@ func CreateContainers(app Application) []v12.Container {
 	for key, value := range app.Environment {
 		envVars = append(envVars,v12.EnvVar{ Name: key, Value: value })
 	}
+	
+	var volumeMounts []v12.VolumeMount
```

Comment:
This could also be preinitalised using `make` with the len of `app.Volumes`, I suppose?